src/schemas/40-10007/schema.js

```diff
@@ -119,8 +118,7 @@ definitions.date = {
 
 definitions.email = {
   type: 'string',
-  maxLength: 50,
-  pattern: '[a-zA-Z0-9_.+-]+@[a-zA-Z0-9_+-]+\\.[a-zA-Z]+'
+  format: 'email'
```

Comment:
I'm not sure we should change this. It may need to be different from the default email format to match a requirement of the backend api.

This also isn't necessary to fix the validation error message on the front end.

Reply:
Thanks for pointing that out @jbalboni, this has been reverted.